DNN Platform/Library/Common/Globals.cs

```diff
@@ -2682,9 +2755,24 @@ public static string LoginURL(string returnUrl, bool overrideSetting)
         /// <param name="overrideSetting">if set to <c>true</c>, show the login control on the current page, even if there is a login page defined for the site.</param>
         /// <param name="portalSettings">The Portal Settings.</param>
         /// <returns>Formatted URL.</returns>
+        [Obsolete("Deprecated in v9.8.0, use the overload that takes IPortalSettings instead, scheduled removal in v10.")]

```

Comment:
Why is this being removed in v10 instead of v11?

Reply:
Addressed in latest commit, thanks.